### ISSUE TITLE
About the discriminator_loss

### ISSUE THREAD

**Issue by xjwxjw** [shekkizh/WassersteinGAN.tensorflow#6]
Hi, 
Much thanks to your excellent work! 
BTW, I am a little bit confused with one line code, which in GAN_models.py line 335: "self.discriminator_loss = tf.reduce_mean(logits_real - logits_fake)" . Shouldn't it be "self.discriminator_loss = tf.reduce_mean(logits_fake - logits_real)"? According to equation (3) in the original paper(Wasserstein GAN,Martin Arjovsky,p.7), it seems that the discriminator_loss should be maximized.

**Comment by shekkizh** [comment]
Hi,
I was following the pytorch implementation of the authors and left the loss definition as such. Whatever formulation you use the discriminator is trying to maximize the difference between real and fake while the generator tries to minimize this difference. Also remember that maximizing a function **f** is the same as minimizing the negative of the function **-f**.

**Comment by xjwxjw** [comment]
Get it, much thanks!
